languages/csharp/Bitwarden.Sdk/Bitwarden.Sdk.csproj

```diff
@@ -0,0 +1,71 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+<PropertyGroup>
+  <TargetFramework>net6.0</TargetFramework>
+  <PackageReadmeFile>README.md</PackageReadmeFile>
+  <ImplicitUsings>enable</ImplicitUsings>
+  <Nullable>enable</Nullable>
+  <AssemblyName>BitwardenSdk</AssemblyName>
+  <PackageId>BitwardenSdk</PackageId>
+  <Title>Bitwarden Sdk</Title>
+  <Authors>Bitwarden Inc.</Authors>
+  <Description>Bitwarden Sdk</Description>
+  <Copyright>Bitwarden Inc.</Copyright>
+  <RepositoryUrl>https://github.com/bitwarden/sdk/tree/master/languages/csharp</RepositoryUrl>
+  <PackageTags>Bitwarden;Sdk;.NET</PackageTags>
+  <Product>BitwardenSdk</Product>
+  <PackageIcon>bitwarden.png</PackageIcon>
+  <RepositoryType>Git</RepositoryType>
+  <RootNamespace>Bitwarden.Sdk</RootNamespace>
+</PropertyGroup>
```

Comment:
```suggestion
  <PackageProjectUrl>https://bitwarden.com/products/secrets-manager/</PackageProjectUrl>
  <PackageLicenseUrl>https://github.com/bitwarden/sdk/blob/master/LICENSE</PackageLicenseUrl>
</PropertyGroup>
```

Reply:
NuGet suggested not adding PackageLicenseUrl just the license packed with the package